workflows/default.yml

```diff
@@ -87,6 +87,8 @@ jobs:
         composer validate
         composer self-update
         composer install
+        ${GITHUB_WORKSPACE}/vendor/bin/phpcs --config-set installed_paths "${GITHUB_WORKSPACE}/vendor/drupal/coder/coder_sniffer"
```

Comment:
phpcs is getting validated as part of the "blt validate" in Validate Step https://github.com/alexku/blt-github-actions/blob/master/scripts/github-actions/validate_app.sh#L5